fg/settings/dummy_secrets.py

```diff
@@ -7,6 +7,8 @@
 # e.g. https://djskgen.herokuapp.com/
 
 #SECRET_KEY = 'xxxxxxxxxxxxxxxxxx'
+#LAB_NAME = 'Lab'
+#TRASH_NAME = 'Trash'
```

Comment:
These should be defined in config.py and have reasonable defaults. That way, if a user just is happy having the `LAB_NAME` be "Lab" and the `TRASH_NAME` be "Trash" they don't have to do any customization.

Reply:
Defined with reasonable defaults